src/components/tearComponent/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
```

Comment:
In order to remain consistent with naming conventions, this file should reflect the component name (pascal case).